x-pack/plugins/security_solution/common/endpoint/index_data.ts

```diff
@@ -92,6 +97,9 @@ export async function indexHostsAndAlerts(
   // Keep a map of host applied policy ids (fake) to real ingest package configs (policy record)
   const realPolicies: Record<string, CreatePackagePolicyResponse['item']> = {};
 
+  await waitForMetadataTransformsReady(client);
+  await stopMetadataTransforms(client);
```

Comment:
The option `startTransform` does not seem to impact the fact that the transforms will **ALWAYS** be stopped. If that is the intent, then can you document this in the function's jsdocs. It could lead to issues if the caller of this function does not realize that and is left with the transforms being stopped. Maybe add a `stopTransform: boolean` option as well?

Reply:
thanks for review and feedback.

this is intentional since we always want to stop the transform prior to indexing so docs get into the dest indices properly. as for starting, since the default is `startTransform = true`, I don't think there's much risk of a consumer accidentally leaving the transform stopped. that said, I think I can make some tweaks and remove the conditional start altogether. will follow up a PR either to document or remove the conditional.